src/app/components/SubmitBar/index.tsx

```diff
@@ -213,6 +219,30 @@ export class SubmitBar extends React.Component<
             <span>SUBMIT</span>
           </button>
         </Tooltip>
+        <Badge
+          anchorOrigin={{
+            horizontal: 'right',
+            vertical: 'top',
+          }}
+          badgeContent={level}
+          color={'secondary'}
+        >
+          <div className={styles.dropdown}>
+            <button className={classnames(styles.customBtn)} title="Quest_Level" id="quest_button">
+              <span className={classnames(styles.icon)}>
+                <FontAwesomeIcon icon={faExclamationCircle} />
+              </span>
+              <span>QUEST LEVEL</span>
+            </button>
+            <div className={styles['dropdown-content']}>
+              <DropDownItem level={'1'} rating={1} openStoryModeModal={openStoryModeModal} />
+              <DropDownItem level={'2'} rating={2} openStoryModeModal={openStoryModeModal} />
+              <DropDownItem level={'3'} rating={3} openStoryModeModal={openStoryModeModal} />
+              <DropDownItem level={'4'} rating={0} openStoryModeModal={openStoryModeModal} />
+              <DropDownItem level={'5'} rating={0} openStoryModeModal={openStoryModeModal} />
```

Comment:
can you get the number of dropdowns and the ratings from props when you will get from an API
so that we can add more levels even after the game is live